components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -124,7 +127,14 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                 setErrorMessage(`Could not subscribe to Stripe. ${error?.message || String(error)}`);
             }
         })();
-    }, [attributionId, location.search]);
+    }, [
+        attributionId,
+        localStorageKey,
+        location.pathname,
+        location.search,
+        pollStripeSubscriptionTimeout,
+        usePublicApiTeamsService,
+    ]);
```

Comment:
Why do we now need all these properties in the effect, when before we didn't?

Reply:
Because we are using them within. If any of them change we want to run the effect again.